index.html

```diff
@@ -73,124 +87,158 @@ <h2 class="proj-title">Tonic</h2>
               <div class="btn-div">
                 <button class="btn" type="submit"> See Project</button>
               </div>
+              </div>
             </div>
-            <div class="second-sec-div">
+            </div>
+            <div class="container2">
+            <div class="second-sec-div2">
               <div>
-                <img class="proj-img" src="images/Snapshoot Portfolio (1).png" alt="picture">
+                <img class="proj-img" src="images//mobile/Snapshoot Portfolio (1).png " alt="picture">
+                <img class="proj-img2" src="images/mobile/Snapshoot Portfolio (3).png" alt="picture">
               </div>
+             <div class="B">
               <div>
                 <h4 class="proj-title">Multi-Post <br>
                   Stories </h4>
+                  <h4 class="proj-title2">Multi-Post
+                    Stories </h4>
+
               </div>
               <div class="proj-intro">
                 <span class="intro-1">CANOPY</span>
-                <img class="dot" src="images/Counter.png" alt="dot">
+                <img class="dot" src="images/mobile/Counter.png" alt="dot">
                 <span class="intro-2">Back end Dev</span>
-                <img class="dot" src="images/Counter.png" alt="dot">
+                <img class="dot" src="images/mobile/Counter.png" alt="dot">
                 <span class="intro-3">2015</span>
               </div>
               <p class="p-descrip">A daily selection of privately personalized reads; no accounts or sign-ups required.
               </p>
+              <p class="p-descrip2">   
+                Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+              </p>
+            
               <ul class="proj-lan">
                 <li class="lan">HTML</li>
+                <li class="lan">Ruby</li>
                 <li class="lan"> CSS</li>
```

Comment:
- [ ] please also make sure the project cards description are aligned properly currently they are improperly aligned and looks different from the original design please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230735095-32636acb-9924-4c0a-ad72-c711d0d1f9d8.png)

figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230735049-d41dbe27-3566-4fa1-9e17-d299411daae9.png)

please try to fix these for all the project cards